commit-message-hook.sh

```diff
@@ -7,9 +7,15 @@ fi
 # Include any branches for which you wish to disable this script
   BRANCHES_TO_SKIP=(master develop staging test)
 
+#Error message to be returned if branch name is not named properly
+ERROR="Branch name is not named properly, please see wiki for formating: https://github.com/space-concordia-robotics/robotics-prototype/wiki/Work-Flow"
```

Comment:
The variable should be renamed to something a bit more significant than just "error", because perhaps we may add more errors and "error" is very vague

Reply:
Also the link doesn't actually lead anywhere, seems like the page hasn't been created yet.

Reply:
@NichHarris @vashmata renamed the page so the link should be updated to `https://github.com/space-concordia-robotics/robotics-prototype/wiki/Git-Workflow-and-Conventions`

Reply:
> Wouldn't that break all the other links to the page you just renamed?
> […](#)
> On Wed, Oct 2, 2019, 12:13 AM PeterGhimself, ***@***.***> wrote: *@PeterGhimself* commented on this pull request. ------------------------------ In commit-message-hook.sh <[#179 (comment)](https://github.com/space-concordia-robotics/robotics-prototype/pull/179#discussion_r330365932)> : > @@ -7,9 +7,15 @@ fi # Include any branches for which you wish to disable this script BRANCHES_TO_SKIP=(master develop staging test) +#Error message to be returned if branch name is not named properly +ERROR="Branch name is not named properly, please see wiki for formating: https://github.com/space-concordia-robotics/robotics-prototype/wiki/Work-Flow" @NichHarris <https://github.com/NichHarris> @vashmata <https://github.com/vashmata> renamed the page so the link should be updated to https://github.com/space-concordia-robotics/robotics-prototype/wiki/Git-Workflow-and-Conventions — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <#179?email_source=notifications&email_token=AEOB46P2ZRZZNYMTH4OFMITQMQNVVA5CNFSM4I3IDOC2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCGSLC5Y#discussion_r330365932>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AEOB46PFLY2W5XNXNB3CRVLQMQNVVANCNFSM4I3IDOCQ> .

You mean all the other links existing somewhere in this repo? Yeah I suppose if they were not taken care of.